docs/current/quickstart/snippets/caching/index.mjs

```diff
@@ -0,0 +1,45 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(async (client) => {
+
+  // highlight-start
+  // create a cache volume
+  const nodeCache = client.cacheVolume("node")
+
+  // use a node:16-slim container
+  // mount the source code directory on the host
+  // at /src in the container
+  // mount the cache volume to persist dependencies
+  const source = await client.container()
+    .from("node:16-slim")
+    .withMountedDirectory('/src', client.host().directory('.', { exclude: ["node_modules/", "ci/"] }))
```

Comment:
see :
https://github.com/dagger/dagger/pull/4391/files#r1087936349 and https://github.com/dagger/dagger/pull/4391/files#r1087939838 regarding directory path